crates/biome_grit_formatter/src/grit/auxiliary/language_declaration.rs

```diff
@@ -13,7 +13,8 @@ impl FormatNodeRule<GritLanguageDeclaration> for FormatGritLanguageDeclaration {
             language_token,
             name,
             flavor,
-            semicolon_token,
+            // Exclude the semicolon token
+            semicolon_token: _,
```

Comment:
If you want to not print something, use `format_remove`.

Our infrastructure keeps track of all tokens and nodes, and not printing all of them is an error.

Reply:
Thanks, fixed!